sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/models/KeyVaultSettingType.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.administration.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+/**
+ * Defines values for {@link KeyVaultSettingType}.
+ */
+public final class KeyVaultSettingType extends ExpandableStringEnum<KeyVaultSettingType> {
```

Comment:
This isn't consistent with our SDK. `KeyVAultRole*` classes use `RoleType`, `KeyVaultKey` uses `KeyType`, etc. This should probably just be `SettingType`.

Reply:
In Java we use `KeyVaultRoleType` in the Key Vault Administration library, which is sadly inconsistent with what we do in Key Vault Keys (`KeyType` instead of `KeyVaultKeyType`). I will talk about this with our architects. As for myself, I think we should stick to what we've already done for the Administration library.